moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -159,6 +159,7 @@ void MotionPlanningFrame::sceneScaleChanged(int value)
     {
       if (ps->getWorld()->hasObject(scaled_object_->id_))
       {
+        const moveit::core::FixedTransformsMap subframes = scaled_object_->subframe_poses_;  // Keep subframes
```

Comment:
[nit] Do you need to copy this to a local if it is const?  Could this be a `const&`?

Or maybe it shouldn't be const if you are planning on modifying it when you scale it?

Reply:
:+1: